addons/stock_account/tests/test_stockvaluation.py

```diff
@@ -3602,3 +3652,32 @@ def test_at_date_average_1(self):
         self.assertEqual(self.product1.with_context(to_date=Datetime.to_string(date1)).value_svl, 100)
         self.assertEqual(self.product1.with_context(to_date=Datetime.to_string(date2)).quantity_svl, 5)
         self.assertEqual(self.product1.with_context(to_date=Datetime.to_string(date2)).value_svl, 50)
+
+    def test_standard_price_fifo_1(self):
+        product = self._get_new_fifo_product()
+        self._make_in_move(product, 3, unit_cost=17)
+        self._make_in_move(product, 1, unit_cost=23)
+        self._make_out_move(product, 3)
+        self.assertEqual(product.standard_price, 23)
+
+    def test_standard_price_fifo_2(self):
+        product = self._get_new_fifo_product()
+        self._make_in_move(product, 5, unit_cost=17)
+        self._make_in_move(product, 1, unit_cost=23)
+        self._make_out_move(product, 4)
+        self.assertEqual(product.standard_price, 17)
+
+    def test_standard_price_fifo_3(self):
+        """Standard price must be set on move in if no product and if first move."""
+        product = self._get_new_fifo_product()
+        self._make_in_move(product, 5, unit_cost=17)
+        self._make_in_move(product, 1, unit_cost=23)
+        self.assertEqual(product.standard_price, 17)
+        self._make_out_move(product, 4)
+        self.assertEqual(product.standard_price, 17)
+        self._make_out_move(product, 1)
+        self.assertEqual(product.standard_price, 23)
+        self._make_out_move(product, 1)
+        self.assertEqual(product.standard_price, 23)
+        self._make_in_move(product, 1, unit_cost=77)
+        self.assertEqual(product.standard_price, 77)
```

Comment:
can you rename these tests

test_fifo_standard_price_upate_1/2/3 and place them just after test_fifo_edit_done_move2 ?